src/main/java/frc/robot/subsystems/Dashboard.java

```diff
@@ -0,0 +1,335 @@
+        // _autoStartPositionChooser.addOption("Substation Side",
+        // DrivePosition.SubstationStart);
+        // _autoStartPositionChooser.addOption("Wall Side", DrivePosition.WallStart);
+        // autonomousLayout.add("Start Position", _autoStartPositionChooser)
+        // .withPosition(0, 1)
+        // .withSize(1, 1)
+        // .withWidget(BuiltInWidgets.kComboBoxChooser);
+
+        var outerLayout = SettingsTab.getLayout("Settings", BuiltInLayouts.kGrid).withPosition(0, 0).withSize(40, 10).withProperties(Map.of("Number of columns", 6, "Number of rows", 1, "Label position", "TOP"));
+
+        var driveSettings = outerLayout.getLayout("Drive Subsystem", BuiltInLayouts.kList).withPosition(0, 0).withSize(8, 10).withProperties(Map.of("label position", "LEFT"));
+        _flOffset = driveSettings.add("FL Offset", 0.0).withPosition(0, 0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _frOffset = driveSettings.add("FR Offset", 0.0).withPosition(2, 0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _blOffset = driveSettings.add("BL Offset", 0.0).withPosition(0, 2).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _brOffset = driveSettings.add("BR Offset", 0.0).withPosition(2, 2).withWidget(BuiltInWidgets.kTextView).getEntry();
+
+        var IntakeSetting = outerLayout.getLayout("Intake", BuiltInLayouts.kList).withPosition(1, 0).withSize(7, 10).withProperties(Map.of("label position", "LEFT"));
+        _IntakeInSpeed  = IntakeSetting.add("Intake In Speed", 0.0).withPosition(0, 0).withWidget(BuiltInWidgets.kTextView).withPosition(0, 0).getEntry();
+        _IntakeOutSpeed = IntakeSetting.add("Intake Out Speed", 0.0).withWidget(BuiltInWidgets.kTextView).withPosition(0, 1).getEntry();
+
+        var climbSetting = outerLayout.getLayout("Climber", BuiltInLayouts.kList).withPosition(2, 0).withProperties(Map.of("label position", "LEFT")).withSize(7, 10);
+        _climberleftOffset  = climbSetting.add("Left Offset", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _climberRightOffset = climbSetting.add("Right Offset", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _climberMinHeight   = climbSetting.add("Arm Minimum Height", 0.0).withPosition(0, 0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _climberMaxHeight   = climbSetting.add("Arm Maximum Height", 0.0).withPosition(2, 0).withWidget(BuiltInWidgets.kTextView).getEntry();
+
+        var noteSetting = outerLayout.getLayout("Notepath", BuiltInLayouts.kList).withPosition(3, 0).withSize(8, 10).withProperties(Map.of("label position", "LEFT"));
+        _NoteShootSpeed     = noteSetting.add("Note Shoot Speed", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _PickupIntakeSpeed  = noteSetting.add("Pickup Intake Speed", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _ShooterIntakeSpeed = noteSetting.add("Shooter Intake Speed", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
```

Comment:
1. Updated tab name
2. Updated layout name
3. Updated variable names
4. Removed `withSize`
5. Use constants for defaults

```suggestion
        var notepathSettings = outerLayout.getLayout("Notepath", BuiltInLayouts.kList).withPosition(3, 0).withProperties(Map.of("label position", "LEFT"));
        _notepathShootSpeed = notepathSettings.add("Note Shoot Speed", Constants.Notepath.NOTEPATH_FEED_PERCENT_OUTPUT).withWidget(BuiltInWidgets.kTextView).getEntry();
        _pickupIntakeSpeed  = notepathSettings.add("Pickup Intake Speed", Constants.Notepath.NOTEPATH_INTAKE_PICKUP_PERCENT_OUTPUT).withWidget(BuiltInWidgets.kTextView).getEntry();
        _shooterIntakeSpeed = notepathSettings.add("Shooter Intake Speed", Constants.Notepath.NOTEPATH_SHOOTER_PICKUP_PERCENT_OUTPUT).withWidget(BuiltInWidgets.kTextView).getEntry();
```